src/index.js

```diff
@@ -67,76 +63,32 @@ class Griddle extends Component {
     super(props);
 
     const {
-      plugins=[],
-      data,
-      children:rowPropertiesComponent,
-      events={},
-      sortProperties={},
-      styleConfig={},
-      pageProperties:importedPageProperties,
-      components:userComponents,
-      renderProperties:userRenderProperties={},
-      settingsComponentObjects:userSettingsComponentObjects,
       storeKey = 'store',
-      reduxMiddleware = [],
-      ...userInitialState
     } = props;
 
-    const rowProperties = getRowProperties(rowPropertiesComponent);
-    const columnProperties = getColumnProperties(rowPropertiesComponent);
+    const { initialState, reducers, reduxMiddleware } = init.call(this, {
+      reducers: dataReducers,
+      components,
+      settingsComponentObjects,
+      selectors,
+      styleConfig: defaultStyleConfig,
 
-    //Combine / compose the reducers to make a single, unified reducer
-    const reducers = buildGriddleReducer([dataReducers, ...plugins.map(p => p.reducer)]);
-
-    //Combine / Compose the components to make a single component for each component type
-    this.components = buildGriddleComponents([components, ...plugins.map(p => p.components), userComponents]);
-
-    this.settingsComponentObjects = Object.assign({}, settingsComponentObjects, ...plugins.map(p => p.settingsComponentObjects), userSettingsComponentObjects);
-
-    this.events = Object.assign({}, events, ...plugins.map(p => p.events));
-
-    this.selectors = plugins.reduce((combined, plugin) => ({ ...combined, ...plugin.selectors }), {...selectors});
-
-    const mergedStyleConfig = _.merge({}, defaultStyleConfig, ...plugins.map(p => p.styleConfig), styleConfig);
-
-    const pageProperties = Object.assign({}, {
+      pageProperties: {
```

Comment:
It looks like the `pageProperties` and `sortProperties` properties were removed. Can this be re-added?

Reply:
As far as I could/can tell, there's actually nothing special about `pageProperties` and `sortProperties` relative to any other "extra" properties that are treated as `initialState`: note tests don't change in https://github.com/GriddleGriddle/Griddle/pull/757/commits/d69275ab85df6a6c60d17ae1818e5e9a45d4612a and https://github.com/GriddleGriddle/Griddle/pull/757/commits/0cbc7d8e14ff64d84f160734885824d4f0c69bbf.